### ISSUE TITLE
Downloaded playlists

### ISSUE THREAD

**Issue by almirsantos99** [spotify/web-api#973]
I don't even knew spotify have more than a ios and android app lol, but i have a suggestion. Why did you guy doesn't create a tab like "my musics" but with "downloaded playlists"? This would be so much helpful, i have something about 50 playlists, search on those to find a downloaded playlist is so much anoying, much more when i trying to find a cool playlist while driving.

**Comment by 0xNF** [comment]
The Ideas section of the Spotify Community site would be a better place to post this:  

https://community.spotify.com/t5/Ideas/ct-p/newideas

**Comment by arirawr** [comment]
Hey @almirsantos99 ! This issue tracker is specifically for the Spotify Web API. As @0xNF mentioned, you should post in the Spotify Community :)

**Comment by riseofthethorax** [comment]
you guys are a bunch of total API nazis, why don't you have rss feeds for users who want to make their playlists public puting all all data behind an authorization interface makes me want to cancel my membership and use youtube-dl to scrape youtube for tunes.. hey I'll give users a bookmarklet that turns youtube playlists and search results into scripts for youtube-dl. I even have scripts that will adjust volume levels of inadequately recorded songs using ffmpeg and sox, and use a method of spiltting songs by using silence detection using ffmpeg's interface, besides open source solutions make capitalistic solutions look like jokes. If people want to discover new music I can help them scrape billboard, discog, and youtube for a way to extract this data using bookmarklets. On my site I can scrape youtube for video codes using CURL in PHP or using bookmarlets, it's how I have compiled playlists on my own website from all of youtubes playlists, its how I got 8000 country western tunes, 5300 60's tunes, 6000 80s pop songs, etc.. It would't take any user long to use bookmarklets to create download scripts using youtube-dl based on youtubes hundreds of playlists, and should nearly destroy your businesz. All I want is a public API..
